freqtrade/rpc/local_rest_server.py

```diff
@@ -0,0 +1,83 @@
+import threading
+import logging
+import json
+
+from flask import Flask, request
+from flask_restful import Resource, Api
+from json import dumps
+from freqtrade.rpc.rpc import RPC
+
+
+logger = logging.getLogger(__name__)
+
+
+class Daily(Resource):
```

Comment:
I don't think we need to subclass Resource in our case. The app.route decorator should do it which much less boilerplate
```
@app.route('/daily', methods=['GET'])
def daily():
    timescale = request.args.get('timescale')
    ...

@app.route('/start', methods=['POST'])
def start():
    ...

...
```

We can bundle all those routes in one file, where freqtrade and _config gets initialized once.

Reply:
Moved to routes as suggested - added / stop and start also